schemas/get-aws-state-response.yml

```diff
@@ -0,0 +1,6 @@
+id:       http://schemas.taskcluster.net/aws-provisioner/v1/get-aws-state-response.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:                      "Get AWS State Response"
+description: |
+  All of the launch specifications for a worker type
+type:                       object
```

Comment:
Should this be array of objects?
IMO, we don't have to declare what properties the objects have, if we just say that they are in a format compatible with the EC2 format from aws-sdk. 

Note, this is a low priority end-point. So another alternative is to just add in the docs a note that says `**This API end-point is experimental and may be subject to change without warning.**`.
Then we don't have to feel bad about changing it..

Note, if you feel like it, I filed bug 1144261 for adding an `stability: 'experimental'` option to `api.declare` this could be useful in many cases. We could probably use node.js stability levels (granted I'm not sure I ever want to promise stable :)
Please share thoughts: https://bugzilla.mozilla.org/show_bug.cgi?id=1144261